domains/client/block-preprocessor/src/lib.rs

```diff
@@ -292,19 +292,6 @@ where
                 )));
             }
 
-            let is_legal_tx = self
-                .client
-                .runtime_api()
-                .check_transaction_validity(at, &extrinsic, domain_block_number, at)?
-                .is_ok();
-
-            if !is_legal_tx {
```

Comment:
Plz reflect the change of checking order here in `InvalidBundleType::checking_order`